src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -15,73 +17,203 @@
 public class SulWowza extends ModuleBase
 {
 
-    public String defaultUrl = "http://localhost:3000/";
-    protected URL baseAuthUrl;
+    public static final String DEFAULT_STACKS_TOKEN_VERIFICATION_BASEURL = "http://localhost:3000";
+    static String stacksTokenVerificationBaseUrl;
 
-    // invoked when a Wowza application instance is started;
-    // defined in the IModuleOnApp interface
+    /** invoked when a Wowza application instance is started;
+     * defined in the IModuleOnApp interface */
     public void onAppStart(IApplicationInstance appInstance)
     {
-        String stacksURL = appInstance.getProperties().getPropertyStr("stacksURL", defaultUrl);
+        String stacksURL = appInstance.getProperties().getPropertyStr("stacksURL", DEFAULT_STACKS_TOKEN_VERIFICATION_BASEURL);
+        stacksTokenVerificationBaseUrl = stacksURL;
         try
         {
-            baseAuthUrl = new URL(stacksURL);
-            getLogger().info("Initialized " + this.getClass().getSimpleName() + " at " + baseAuthUrl);
+            new URL(stacksTokenVerificationBaseUrl);
+            getLogger().info(this.getClass().getSimpleName() + " stacks token verification baseUrl is " + stacksTokenVerificationBaseUrl);
         }
         catch (MalformedURLException err)
         {
-            getLogger().error("Unable to initialize " + this.getClass().getSimpleName() + " module", err);
+            getLogger().error(this.getClass().getSimpleName() + " unable to initialize module due to bad stacksURL: ", err);
         }
     }
 
-    // Invoked when an HTTP MPEGDash Streaming session is created;
-    // defined in IModuleOnHTTPMPEGDashStreamingSession module interfaces
+    /** Invoked when an HTTP MPEGDash Streaming session is created;
+     * defined in IModuleOnHTTPMPEGDashStreamingSession module interfaces */
     public void onHTTPMPEGDashStreamingSessionCreate(HTTPStreamerSessionMPEGDash httpSession)
     {
-        getLogger().info("Tommy: SulWowza onHTTPMPEGDashStreamingSessionCreate: " + httpSession.getSessionId());
+        getLogger().info(this.getClass().getSimpleName() + " onHTTPMPEGDashStreamingSessionCreate: " + httpSession.getStreamName());
```

Comment:
Let's say that something went terribly wrong during initialization, and stacksTokenVerificationBaseUrl is null. Could we add a test for this case? I assume that if we know that the verification process will fail, there's no point in going ahead from here. Maybe just reject the session and output an error message to the log? The same goes for onHTTPCupertinoStreamingSessionCreate() below.